waterlog/src/Components/DailyWastage.js

```diff
@@ -1,34 +1,53 @@
 import React from 'react';
 import { Line, defaults } from 'react-chartjs-2';
+import { Globals } from './../Globals';
 
 const DailyWastageComponent = (props) => {
+  let labels = props.props.dataPoints.map(a => (new Date(a.x).getHours()+":00"));
+  let datas = props.props.dataPoints.map(a => a.y);
+  var sum = datas.reduce((a, b) => a + b, 0);
```

Comment:
Why mix let and var? I would just use const and let.

Reply:
Switched them all over to var.

Reply:
I would have prefer let and const but ok.